src/main/java/com/minecolonies/core/network/messages/client/colony/PlaySoundForCitizenMessage.java

```diff
@@ -150,7 +150,7 @@ public PlaySoundForCitizenMessage(final int entityID, final SoundEvent event, fi
     @Override
     public void toBytes(final FriendlyByteBuf buf)
     {
-        buf.writeResourceLocation(ForgeRegistries.SOUND_EVENTS.getKey(this.soundEvent));
+        buf.writeResourceLocation(this.soundEvent.getLocation());
```

Comment:
Does this solve the issue with "Sounds"?

Reply:
ye

Reply:
we're actually checking the registry for the existence of the sound event, but the registry doesnt know the actual sound event instance, so that fails then